porcupine/plugins/run/no_terminal.py

```diff
@@ -258,6 +278,19 @@ def stop_executor(self, junk_event: object = None, *, quitting: bool = False) ->
         if self.executor is not None:
             self.executor.stop(quitting=quitting)
 
+    def pause_resume_executor(self, junk: object = None) -> None:
+        if sys.platform == "win32" or self.executor is None:
+            return
```

Comment:
Mypy's unreachable code logic doesn't understand early-`return` very well: even though `mypy --platform win32` understands that the if statement will always execute, it doesn't realize that the code after the `return` is still reachable on other platforms.

To work around this, we can just avoid early returns:

```
if sys.platform != "win32" and self.executor is not None:
    if self.executor.paused:
        ...
```